src/forms/TimeInput.jsx

```diff
@@ -10,6 +17,15 @@ class TimeInput extends React.Component {
 	constructor(props) {
 		super(props);
 
+		this.state = {
+			hours: props.hours || '01',
+			minutes: props.minutes || '00',
+			meridian: props.meridian || props.isMilitaryTime ? false : 'PM'
```

Comment:
so this `meridian`/`isMilitaryTime` prop in a way would be dictated by locale right? - since some use 24hr time and some 12hr.
it'd be nice if we could find a way to pass in locale and use that to dictate all of this.

`props.is24Hr` might be better than military?

Reply:
I thought about doing that, but since all the locale stuff is in mup-web, I was thinking we could check locale in there and pass a true or false from that information. If that doesn't make sense, I'm happy to put the logic into the component

Agreed on the prop name change though 👍